pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -704,6 +704,12 @@
     )
     private boolean isAllowAutoUpdateSchemaEnabled = true;
 
+    @FieldContext(
+            category = CATEGORY_SERVER,
+            doc = "Whether ConcurrentLongLongPairHashMap supports automatic shrinking," +
+                    "the default is false, which means automatic shrinking is not supported.")
+    private boolean autoShrink = false;
```

Comment:
we shouldn't add this knob.
and if we really want the name should be more narrow and point exactly to the thing that we are changing

Reply:
Don't need this configuration? You still need to modify the configuration name, for example: autoShrinkForConcurrentLongLongPairHashMap?

Reply:
it is only for pendingAcks at the moment, so the name should reflect it.
it is not for every ConcurrentLongLongPairHashMap

Reply:
like this?
autoShrinkForPendingAcks

Reply:
Fixed:
rename autoShrink to autoShrinkForConsumerPendingAcksMap;